test/testdata/fixtures/multi-dc-stargate/k8ssandra.yaml

```diff
@@ -1,3 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cassandra-config
+data:
+  cassandra.yaml: |
+    concurrent_reads: 32
+    concurrent_writes: 32
```

Comment:
Probably bad for performance to have so many readers and writers in a restricted CPU core count.

Reply:
I don't think it matters much because we're not doing anything that's heavy on performance, but I'll lower them.